Routes/backend/general.php

```diff
@@ -28,9 +28,15 @@ function () {
         //------------------------------------------------
         Route::post( '/signin/generate/otp', 'PublicController@postGenerateOTP' );
         //------------------------------------------------
+        Route::post( '/resendSecurityOtp/post', 'PublicController@signinResendSecurityOtp' )
```

Comment:
@we-vikram-wri231 even route path naming convention is not correct, what did you check?